src/courses/tests/test_models.py

```diff
@@ -277,6 +277,17 @@ def test_default_object_created(self):
         """ A data migration should make a default object for this model """
         self.assertTrue(Course.objects.filter(title="Default").exists())
 
+    def test_model_protection(self):
+        """ 
+            quick test to see if model deleteion is protected when trying to delete progmatically
```

Comment:
Two typos here (is spell check turned on in your IDE? Also, be more specific about what you are actually testing.  It's not clear to me from the docstring what you are testing specifically, which model? what's being deleted? what should happen?

Reply:
fixed

Reply:
didnt have a spell check will install one